app/src/main/java/com/tohami/newsapi/ui/news/NewsFragment.java

```diff
@@ -0,0 +1,105 @@
+package com.tohami.newsapi.ui.news;
```

Comment:
There should be a dedicated package for newsListing with the following classes,
fragment, viewmodel, adapter, repository

Reply:
done